File: 0801_0900/0803/0803.py
with open("input.txt", "r") as f, open("output.txt", "w") as q:
    a, b, k = (int(x) for x in f.readline().split())
    for i in range(k):
        a = a % b * 10
    q.write(str(a // b))

File: 0801_0900/0803/0803.c
#include <stdio.h>

int main()
{
	FILE *f = fopen("input.txt", "r"), *q = fopen("output.txt", "w");
	int a, b, k;

	fscanf(f, "%d%d%d", &a, &b, &k);
	for (int i = 0; i < k; ++i)
		a = a % b * 10;
	fprintf(q, "%d", a / b);
	return 0;
}
